clearwater-infrastructure/usr/share/clearwater/bin/generic_create_diameterconf

```diff
@@ -21,8 +21,9 @@ port=$4
 sec_port=$5
 acl_required=$6
 check_dh=$7
+tw_timer=$8
 
-if [[ $# != 7 ]]
+if [[ $# != 8 ]] || [[ $tw_timer -lt 6 ]]
```

Comment:
I would split these into to separate tests. That way if the second test fails you can print a more useful error message about what was wrong.